### DISCUSSION TITLE
Custom Token parser

### DISCUSSION THREAD

**Discussion by erik-zicht** [VincentLanglet/Twig-CS-Fixer#89] [General]
In response to https://github.com/VincentLanglet/Twig-CS-Fixer/issues/61 and the remark `We could add a config to add custome token parser.`  I am having the same errors ("File is invalid...") due to a custom Token we have created ourselves. 

Is it already possible to add custom tokens to the configuration?

**Comment by VincentLanglet** [comment]
Yes, you now have a `addTokenParser` on the Config.
https://github.com/VincentLanglet/Twig-CS-Fixer/blob/main/src/Config/Config.php#L110

It's not yet added to the readme
